tensorflow/lite/micro/tools/make/targets/cortex_m_generic_makefile.inc

```diff
@@ -0,0 +1,156 @@
+# Generic Makefile target for ARM Cortex M builds.
+# REQUIRED:
+#   - TAGS: armclang or armgcc
+# For Cortex-M55, ARM Compiler 6.14 or later is required.
+# Example:
+# make -f tensorflow/lite/micro/tools/make/Makefile TAGS="cmsis-nn armclang"  TARGET=cortex-m55-generic microlite
```

Comment:
I recognize that we haven't set a good example of documenting the Makefile usage. But this is likely a TARGET that will be used quite broadly. It would be great if you could move this documentation to the README that @ml-0 had started and provide more variations of the different ways to invoke this particular target.

Reply:
I will update the README.